lib/ansible/module_utils/network/nxos/nxos.py

```diff
@@ -194,6 +194,9 @@ def load_config(self, config, return_error=False, opts=None):
         try:
             responses = connection.edit_config(config)
             out = json.loads(responses)[1:-1]
+            for o in out:
+                if "at '^' marker" in o and "Invalid" in o:
+                    self._module.fail_json(msg=o)
```

Comment:
imo instead of searching for error string it should be added in [nxos terminal plugin](https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/terminal/nxos.py#L35) `terminal_stderr_re `

Reply:
fixed.